test/Microsoft.AspNetCore.Mvc.Formatters.Xml.Test/XmlDataContractSerializerInputFormatterTest.cs

```diff
@@ -301,9 +302,7 @@ public void SetMaxDepth_ThrowsWhenMaxDepthIsBelowOne()
             // Arrange
             var expectedException = TestPlatformHelper.IsMono ? typeof(SerializationException) :
                                                                 typeof(XmlException);
-            var expectedMessage = TestPlatformHelper.IsMono ?
-                "Expected element 'TestLevelTwo' in namespace '', but found Element node 'DummyClass' in namespace ''" :
-                "The expected encoding 'utf-8' does not match the actual encoding 'utf-16LE'.";
+            var expectedMessage = "The expected encoding 'utf-8' does not match the actual encoding 'utf-16LE'.";
```

Comment:
Why are we checking for a specific framework message? A couple of `Assert.Contains` assertions for the two encoding names should be enough, right?